src/index.js

```diff
@@ -63,7 +76,9 @@ const expandShorthands = (tr) => {
  * @returns {HigherOrderComponent}
  */
 export default (transformations = []) => {
-  const transformsList = castArray(transformations).map(expandShorthands)
+  const transformsList = defaultTransformations.concat(
```

Comment:
This is not that readable, what about creating `userTransformations`? Also default need not to be expanded.